package.json

```diff
@@ -40,7 +40,8 @@
     "is-npm": "^1.0.0",
     "latest-version": "^2.0.0",
     "semver-diff": "^2.0.0",
-    "xdg-basedir": "^2.0.0"
+    "xdg-basedir": "^2.0.0",
+    "lazy-req" : "^1.1.0"
```

Comment:
If we _do_ eventually merge this, we should put this dependency in alphabetical order. Typically npm will do this for us via `npm install --save lazy-req`.